src/Nethermind/Nethermind.Evm.Test/TransactionProcessorTests.cs

```diff
@@ -58,7 +58,7 @@ public void Setup()
         _stateProvider.Commit(_specProvider.GenesisSpec);
         _stateProvider.CommitTree(0);
 
-        CodeInfoRepository codeInfoRepository = new();
+        CodeInfoRepository codeInfoRepository = new(1);
```

Comment:
Why are we passing `1` as `chainId` instead of `_specProvider.ChainId`?

Reply:
wrong PR